ci/run.sh

```diff
@@ -122,6 +125,10 @@ case "$TARGET" in
     grep "^PASSED .* tests" /tmp/out
     ;;
 
+  x86_64-linux-android)
+    $CARGO_TARGET_DIR/$TARGET/debug/libc-test
+    ;;
```

Comment:
I think this can be omitted right to fall down to the fall through at the bottom?